wd_helper.au3

```diff
@@ -490,14 +490,14 @@ EndFunc   ;==>_WD_LastHTTPResult
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_GetFrameCount
-; Description ...: This will return how many frames/iframes are in your current window/frame.
+; Description ...: This will return how many frames/iframes are in your current context (window/frame/document).
```

Comment:
What's the difference between window and document in this context? Are they truly different?

Reply:
in JavaScript you can use something like this: `window.frames[i].document.body.style.background = "red";` and that mean when you are inside of any frame then you are on Document. And thus my recent commit - please check them.